### ISSUE TITLE
Can't unlock desktop - Cinnamon 3.2

### ISSUE THREAD

**Issue by Draconicrose** [linuxmint/Cinnamon#6002]
Whether the desktop locks through timeout or through a shortcut, when I try to unlock it again there's no UI for it. I've tried inputting the password anyway and pressing enter, but I'm unable to return to the desktop and resume my session. I've been using ctrl + alt + backspace to log back in but of course my session is gone.

**Comment by JosephMcc** [comment]
You need to provide more info, such as exact version of cinnamon-screensaver, cinnamon, and at the very least the distro you are running on.

By no UI you mean you can see the darkened background but just have no entry for the password?

**Comment by Draconicrose** [comment]
cinnamon-screensaver 3.2.3
Cinnamon 3.2.1
Linux Mint 18

![Blank unlock desktop screen](http://i65.tinypic.com/ipv77r.jpg)

Yes, the left monitor is supposed to be off.

**Comment by fatalcoder** [comment]
Similar here. I am able to unlock though.
```
DELL XPS 13 x86_64 HiDPI 
Fedora 25
cinnamon 3.2.1
cinnamon-screensaver 3.2.0
```

![zrzut ekranu z 2016-11-27 10-59-51](https://cloud.githubusercontent.com/assets/3192656/20647317/76f413f8-b491-11e6-9a4d-069775a2067c.png)

**Comment by leigh123linux** [comment]
@fatalcoder Your issue is easy to fix, gtk3-3.22.4-1.fc25 fixes the scaling issue, it's currently being pushed to updates-testing.
Or you could get it from koji (fedora build server).

https://koji.fedoraproject.org/koji/buildinfo?buildID=819121

**Comment by fatalcoder** [comment]
@leigh123linux yup, helped. Thanks a lot, again

**Comment by roman-yagodin** [comment]
Having similar issue here. Sometimes it's OK, but in most cases there are just dimmed screen w/o UI on locked screen. Typing password blindly helps to unlock the screen.

Cinnamon 3.2.7
cinnamon-screensaver 3.2.12
LMDE 2 Betsy x86_64

**Comment by zquestz** [comment]
I am also experiencing this issue. In addition xscreensaver processes never quit after unlock.

**Comment by roman-yagodin** [comment]
I think it's working after recent updates now.

**Comment by zquestz** [comment]
Seems to happen to me far more often when I have 2 displays setup.

**Comment by dg1727** [comment]
I just now had a similar thing happen:  

I'm not using cinnamon-screensaver at all, only xscreensaver.  (In the cinnamon-screensaver settings, which are System Settings > Screensaver > Settings tab > "Screensaver settings" section, the delay is set to Never.)  
I'm also **not** using the cinnamon-screensaver plugin that lets xscreensaver settings be configured from that same System Settings window.  
LMDE 2 "Betsy" Cinnamon 64-bit.  I have installed mintUpdate's level 1, 2, 3 (but I think only one level 4 & no level 5) updates from jessie-backports.  
xscreensaver package version 5.30-1+deb8u2
`uname -rv` says `3.16.0-4-amd64 #1 SMP Debian 3.16.39-1 (2016-12-30)`
/usr/bin/cinnamon is at version 3.2.7

My recollection of what happened:  
The screen had been locked for several hours, with the screensaver running.  
When I moved the mouse, the screensaver froze and the mouse pointer appeared, but the unlock dialog didn't appear.  
The mouse pointer was restricted to the screen area where the unlock dialog would normally be.  
I tried to type my password but may have made a typo.  The screensaver was still frozen, but the mouse restriction went away and I couldn't get it to come back by moving the mouse or pressing keys on the keyboard.  
Ctrl-Alt-F1 (to bring up a full-screen text-mode console) didn't work!!  
After Ctrl-Alt-F1, I tried typing my password blindly and `sudo -s`, then my password again and `pkill xscreensaver`, but it didn't work.  
SysRq + R, E, I, S, U, B did reboot.  I didn't remember which modifier keys are needed for SysRq, so I just held down Shift, Ctrl, Alt simultaneously.  

I think I've had the screensaver running like that before without the hang, so this would be an intermittent problem.  

Other users have mentioned MDM; I have my desktop set to log me in automatically (this selection was made in the OS installer), so I normally don't think about a display manager.
